src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/PuppetGatlingProjectAction.java

```diff
@@ -68,35 +70,35 @@ public int getLastSuccessfulNumber() {
         return latestSuccessfulBuild.getNumber();
     }
     
-    public Graph<Long> getdashboardGraph() {
-    	return new Graph<Long>(job, MAX_BUILDS_TO_DISPLAY_DASHBOARD) {
+    public Graph<Long> getDashboardGraph() {
+    	return new SimulationGraph<Long>(job, MAX_BUILDS_TO_DISPLAY_DASHBOARD) {
```

Comment:
`SimulationGraph` is just a new subclass that I introduced (I changed `Graph` to be an interface) so that we could draw Graphs of things that don't come directly from gatling.